### DISCUSSION TITLE
Set validity by index for Utf8Array

### DISCUSSION THREAD

**Discussion by Arty-Maly** [jorgecarleitao/arrow2#1366] [Q&A]
I am curious if there is a better way to set a specific index to valid/invalid in a Utf8Array? 

For PrimitiveArray for example I can do `apply_validity` But I don't think there is a method for Utf8Array. What would be an idiomatic way to invalidate an element in an Utf8Array?

**Comment by jorgecarleitao** [comment]
I think it would be a good idea to create one in `Utf8Array`, exactly for the use-case you are looking for!

Would you like to create a PR on it, or would you like me to do it?

**Reply by Arty-Maly** [reply]
I can create it, since I actually need this functionality.
